README.Rmd

```diff
@@ -21,248 +21,58 @@ knitr::opts_chunk$set(
 (ODBC) drivers. This allows for an efficient, easy to setup connection to any
-database with ODBC drivers available, including [SQL
-Server](https://www.microsoft.com/en-us/sql-server/),
-[Oracle](https://www.oracle.com/database), [MySQL](https://www.mysql.com/),
-[PostgreSQL](https://www.postgresql.org/), [SQLite](https://sqlite.org/index.html) and
-others. The implementation builds on the
-[nanodbc](https://nanodbc.github.io/nanodbc/) C++ library.
-
--   [Installation](#installation)
-    -   [Windows](#windows)
-    -   [MacOS](#macos)
-    -   [Linux - Debian / Ubuntu](#linux---debian-ubuntu)
-    -   [R](#r)
--   [Connecting to a Database](#connecting-to-a-database)
-    -   [Connection Strings](#connection-strings)
-    -   [DSN Configuration files](#dsn-configuration-files)
--   [Usage](#usage)
-    -   [Table and Field information](#table-and-field-information)
-    -   [Reading](#reading)
-    -   [Writing](#writing)
-    -   [Querying](#querying)
--   [Benchmarks](#benchmarks)
-    -   [Reading](#reading-1)
-    -   [Writing](#writing-1)
-
-## Installation
-For Unix and MacOS ODBC drivers should be compiled against
-[unixODBC](https://www.unixodbc.org/). Drivers compiled against [iODBC](https://www.iodbc.org/) _may_ also
-work, but are not fully supported.
-
-After installation of the driver manager and driver, you will have to
-register the driver in a [odbcinst.ini](#dsn-configuration-files) file for it
-to appear in `odbc::odbcListDrivers()`.
-
-odbc and its dependencies use C++11 features. Therefore you need [gcc
-4.8](https://gcc.gnu.org/), [clang 3.3](https://clang.llvm.org) or [Rtools
-3.3](https://CRAN.R-project.org/bin/windows/Rtools/) or later.
-
-### Windows
-Windows is bundled with ODBC libraries. However, drivers for each database
-need to be installed separately. Windows ODBC drivers typically include an
-installer that needs to be run and will install the driver to the proper
-locations.
-
-### MacOS
-[homebrew](https://brew.sh/) can be used to easily install database drivers on MacOS.
-
-#### UnixODBC - Required for all databases
-```shell
-# Install the unixODBC library
-brew install unixodbc
-```
-
-#### Common DB drivers
-```shell
-# Microsoft SQL Server ODBC Drivers (Recommended)
-brew tap microsoft/mssql-release https://github.com/Microsoft/homebrew-mssql-release
-brew update
-brew install msodbcsql17 mssql-tools
-
-# SQL Server ODBC Drivers (Free TDS)
-brew install freetds
+database with support for ODBC.
 
-# PostgreSQL ODBC ODBC Drivers
-brew install psqlodbc
-
-# MySQL ODBC Drivers (and database)
-brew install mysql
-
-# SQLite ODBC Drivers
-brew install sqliteodbc
-```
+## Overview
 
-### Linux - Debian / Ubuntu
-[apt-get](https://wiki.debian.org/Apt) can be used to easily install database
-drivers on Linux distributions which support it, such as Debian and Ubuntu.
+The odbc package is one piece of the R interface to databases with support for ODBC:
```

Comment:
I use "database" throughout, but should we transition to using "DBMS"? Or include both and clarify their difference?

Reply:
Yeah, I like DBMS for clarity.

Reply:
You got it🏄 [1f9f52f](https://github.com/r-dbi/odbc/pull/652/commits/1f9f52f2e6cf80ee7f5e91c1126359c1eb5f58b1)